examples/jupyter/bin/marathon_submit.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+curl -k -i -L -X POST -H "Content-type: application/json" "https://$MARATHON/v2/apps" -d@"$@"
```

Comment:
username and password are in the $MARATHON env which is exported, I will make a single script to avoid that.